File: webcaptioner_controller.py
import obspython as obs
from obs_change_notifier import OBSChangeNotifier

client = OBSChangeNotifier(
           obs=obs,
           watched_source='Captions Enabled',
           base_url='http://localhost:4567'
         )

def script_description():
  global client

  return "Subtitle management for " + client.watched_source

def script_load(settings):
  global client

  client.connect('source_activate', '/control', {'enabled': 'true'})
  client.connect('source_deactivate', '/control', {'enabled': 'false'})

File: webcaptioner_controller.lua
local obs = require 'obslua'
local notifier = require 'obs_change_notifier'

source = 'Captions Enabled'

function script_load(settings)
  notifier.configure({
    source = source,
    active_url = {'http://localhost:4567/control', {enabled = 'true'}},
    inactive_url = {'http://localhost:4567/control', {enabled = 'false'}}
  })
end

function script_description()
  return "Subtitle management for " .. source
end
